students/jaemoon/users/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Modle):
```

Comment:
재문님! PR 작성하시기 전에 전체적으로 코드에 문제가 없는지 확인 후에 PR 올려주세요!!
```suggestion
class User(models.Model):
```